v12/storage-dotnet-redundancy-ha-ra-grs/Program.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System.Diagnostics.Tracing;
```

Comment:
Are all of the `using` statements required? It might be nice to put these in a `GlobalUsings.cs` file, and mark each `using` as `global`. Then you don't need them in the `Program.cs` file.